lib/core/cursor.js

```diff
@@ -48,6 +49,13 @@ const Long = BSON.Long;
 var Cursor = function(topology, ns, cmd, options) {
   options = options || {};
 
+  if (typeof ns !== 'string') {
```

Comment:
what is going on here?

Reply:
For the time being `Cursor` supports either passing in an operation, or a namespace + command. Since the `MongoDBNamespace` changes never made it to core, we can infer that if `ns` is not a string then we are calling the "operation" variant of this constructor. This will all go away once retryable reads is fully implemented.